### PR TITLE
Add CONTRIBUTING.md file with guidelines for contributing to FastEmbed

### PR DESCRIPTION
**Pull request by NirantK**
Added a CONTRIBUTING.md for FastEmbed.

Hoping to reduce support workload in the build up to Qdrant Summer of Code

### CODE REVIEW

**Comment by Anush008** [comment]
We could also add
```
    - name: Run pre-commit
      run: pre-commit run --all-files
```
at https://github.com/qdrant/fastembed/blob/add-contributing/.github/workflows/python-tests.yml#L42-L46.

**Comment by NirantK** [comment]
@Anush008 add that to a separate PR as well please?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+* **Provide specific examples to demonstrate the steps**. Include links to files or GitHub projects, or copy/pasteable snippets, which you use in those examples. If you're providing snippets in the issue, use [Markdown code blocks](https://help.github.com/articles/markdown-basics/#multiple-lines).
+* **Describe the behavior you observed after following the steps** and point out what exactly is the problem with that behavior.
+* **Explain which behavior you expected to see instead and why.**
+* **If the problem is related to performance or memory**, include a [call stack profile capture](https://github.com/joerick/pyinstrument) and your observations. 
+
+Include details about your configuration and environment:
+
+* **Which version of FastEmbed are you using?** You can get the exact version by running `python -c "import fastembed; print(fastembed.__version__)"`.
+* **What's the name and version of the OS you're using**?
+* **Which packages do you have installed?** You can get that list by running `pip freeze`
```
Comment (KShivendu) [inline CONTRIBUTING.md:45]: Can we add this in Github issue template instead? That's more likely to be read.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+The best way to learn about the mechanics of FastEmbed is to start working on it. 
+
+### Your First Code Contribution
+Your first code contribution can be small bug fixes:
+1. This PR adds a small bug fix for a single input: https://github.com/qdrant/fastembed/pull/148
+2. This PR adds a check for the right file location and extension, specific to an OS: https://github.com/qdrant/fastembed/pull/128
+
+Even documentation improvements are most welcome:
+1. This PR fixes a README link: https://github.com/qdrant/fastembed/pull/143
```
Comment (KShivendu) [inline CONTRIBUTING.md:63]:
```suggestion
Even documentation improvements and tests are most welcome:
1. This PR fixes a README link: https://github.com/qdrant/fastembed/pull/143
```

**Review by KShivendu — APPROVED** [review]
LGTM

**Pull request merged**